ezyrb/approximation/linear.py

```diff
@@ -41,7 +41,10 @@ def fit(self, points, values):
 
         if as_np_array.ndim == 1 or (as_np_array.ndim == 2
                                      and as_np_array.shape[1] == 1):
-            self.interpolator = interp1d(as_np_array, values, axis=0)
+
+            self.interpolator = interp1d(as_np_array, values, axis=0,
```

Comment:
In this class, it is quite difficult to keep that general layout, mainly because we are internally using two different methods depending on the input dimension!